Assignment2/RESKOM326-fromJSONtoJSON-LD.jsonld

```diff
@@ -0,0 +1,39 @@
+{
+    "@context":{
+        "info": "http://www.mydata/api/general",
+        "rdfs": "https://www.w3.org/2000/01/rdf-schema#",
+        "uni": "https://www.w3.org/2000/01/university#",
+        "Parents name": "info:data/personal/parentsName",
+        "Full name": "rdfs:label",
+        "Hometown": "rdfs:label",
+        "Age": "rdfs:label",
+        "Alma mater": "rdfs:label",
```

Comment:
Syntactically is ok, but you have to define each attribute with a specific vocabulary attribute name because when you want to retrieve, for example your age, if you query "rdfs:label" you will not be able to distinguish what data you ask for. If you have your own vocabulary you can use it to define your attributes e.g. "name":"yourvocab:name", "age":"yourvocab:age", etc.

Reply:
Done